src/ImageSharp/Formats/Tiff/TiffDecoderCore.cs

```diff
@@ -311,17 +265,12 @@ private int CalculateStripBufferSize(int width, int height, int plane = -1)
         /// <param name="stripOffsets">An array of byte offsets to each strip in the image.</param>
         /// <param name="stripByteCounts">An array of the size of each strip (in bytes).</param>
         /// <param name="width">The image width.</param>
-        /// <param name="predictor">The tiff predictor used.</param>
-        private void DecodeStripsPlanar<TPixel>(ImageFrame<TPixel> frame, int rowsPerStrip, uint[] stripOffsets, uint[] stripByteCounts, int width, TiffPredictor predictor)
+        private void DecodeStripsPlanar<TPixel>(ImageFrame<TPixel> frame, int rowsPerStrip, uint[] stripOffsets, uint[] stripByteCounts, int width)
             where TPixel : unmanaged, IPixel<TPixel>
         {
             int stripsPerPixel = this.BitsPerSample.Length;
             int stripsPerPlane = stripOffsets.Length / stripsPerPixel;
-            int bitsPerPixel = 0;
-            foreach (var bits in this.BitsPerSample)
-            {
-                bitsPerPixel += bits;
-            }
+            int bitsPerPixel = this.ChunkyBitsPerPixel; // todo?
```

Comment:
what is this TODO implying? Can you give more context? bitsPerPixel is used below.

Reply:
see https://github.com/SixLabors/ImageSharp/pull/1471#discussion_r542505849